test/functional/test_runner.py

```diff
@@ -463,7 +473,7 @@ def get_next(self):
                     proc.send_signal(signal.SIGINT)
                 if proc.poll() is not None:
                     log_out.seek(0), log_err.seek(0)
-                    [stdout, stderr] = [log_file.read().decode('utf-8') for log_file in (log_out, log_err)]
+                    [stdout, stderr] = [log_file.read().decode('utf-8', 'ignore') for log_file in (log_out, log_err)]
```

Comment:
I'd prefer to keep the throw on non-unicode log files.

Reply:
Same as above.